global-script.lua

```diff
@@ -1888,9 +1912,8 @@ function addSpirit(params)
 end
 function removeSpirit(params)
     SetupChecker.call("removeSpirit", params)
-    getObjectFromGUID(elementScanZones[params.color]).clearButtons()
```

Comment:
where's the code for clearing out these buttons currently live?

Reply:
For some reason, I thought these were the element count buttons and such.  Nope.  They're from Gain a Spirit.

Which removes them on its own.

... but they don't get removed if you click Gain a Spirit, and then choose a spirit not on that list of four.  Re-added this line.